Updates the older Libuuid to LibUUID so CMake does not complain
Author: J. Sassmannshausen
diff --git a/crossguid-0.2.2.orig/CMakeLists.txt b/crossguid-0.2.2/CMakeLists.txt
index 761dbc6..22fcee5 100644
--- a/crossguid-0.2.2.orig/CMakeLists.txt
+++ b/crossguid-0.2.2/CMakeLists.txt
@@ -27,7 +27,7 @@ elseif(APPLE)
 elseif(ANDROID)
 	target_compile_definitions(xg PRIVATE GUID_ANDROID)
 else()
-	find_package(Libuuid REQUIRED)
+	find_package(LibUUID REQUIRED)
 	if (NOT LIBUUID_FOUND)
 		message(FATAL_ERROR
 			"You might need to run 'sudo apt-get install uuid-dev' or similar")
diff --git a/crossguid-0.2.2.orig/cmake/FindLibuuid.cmake b/crossguid-0.2.2/cmake/FindLibUUID.cmake
similarity index 100%
rename from crossguid-0.2.2.orig/cmake/FindLibuuid.cmake
rename to crossguid-0.2.2/cmake/FindLibUUID.cmake
